### PR TITLE
Add Conn() support to all storage drivers.

### LINKED ISSUE #440: [Feature Request] Support for retrieving database connections from storage structures
Provide a way to retrieve the db object from the storage.

Example:
```
store := redis.New()
db := store.DB()
keys := db.keys()
```

Currently theres no way to access the DB from any of the storage drivers, limiting the user options within Fiber.

### PR DESCRIPTION
**Pull request by gaby**
This PR adds support for `Conn()` to all drivers. The main purpose is allow access to the underlying database client from the interface.

Fixes #440

### CODE REVIEW

**Comment by ReneWerner87** [comment]
thanks for that big PR -> will check it soon

**Comment by efectn** [comment]
Wouldn't `Conn()` be more logical than `DB()` 🤔🤔

**Comment by gaby** [comment]
@efectn Agree, I can change it

**Comment by gaby** [comment]
I'm still not sure what to do with the `memory` driver, given that it returns a Map of Entry.

**Comment by gaby** [comment]
Done, all functions have been renamed from `DB()` to `Conn()`

File: arangodb/arangodb_test.go [hunk 1]
```diff
@@ -132,3 +132,7 @@ func Test_ARANGODB_Non_UTF8(t *testing.T) {
 func Test_ARANGODB_Close(t *testing.T) {
 	utils.AssertEqual(t, nil, testStore.Close())
 }
+
+func Test_ARANGODB_DB(t *testing.T) {
```
Comment (efectn) [inline arangodb/arangodb_test.go:136]: Please change Test_*_DB to Test_*Conn_

**Review by efectn — CHANGES_REQUESTED** [review]

File: arangodb/arangodb_test.go (hunk 1 above)
Comment (gaby) [reply arangodb/arangodb_test.go]: Done

File: bbolt/README.md [hunk 2]
```diff
@@ -17,6 +17,7 @@ func (s *Storage) Set(key string, val []byte, exp time.Duration) error
 func (s *Storage) Delete(key string) error
 func (s *Storage) Reset() error
 func (s *Storage) Close() error
+func (s *Storage) Conn() bbolt.DB
```
Comment (efectn) [inline bbolt/README.md:20]: There're some mistakes on READMEs like this. please add pointer if they return pointer

File: README.md [hunk 3]
```diff
@@ -46,6 +46,9 @@ type Storage interface {
 	// Close closes the storage and will stop any running garbage
 	// collectors and open connections.
 	Close() error
+
+	// Returns the underlying DB client
+	Conn() db.Client
```
Comment (efectn) [inline README.md:51]: I think we don't need here because of we dont add `Conn()` into the `Storage` interface.

**Review by efectn — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (gaby) [reply README.md:51]: Done

File: bbolt/README.md (hunk 2 above)
Comment (gaby) [reply bbolt/README.md:20]: Done

Comment (gaby) [reply bbolt/README.md]: The only ones that don't return pointers are `Memory` and `ArangoDB`. I have updated the README for all the other drivers.

**Review by efectn — APPROVED** [review]

**Comment by gaby** [comment]
@efectn @ReneWerner87 Who can approve the workflows to be run? Thanks

**Comment by gaby** [comment]
I'm not sure which field to use for ArangoDB.

**Comment by gaby** [comment]
@efectn @ReneWerner87 I replaced the return with `driver.Client` since from the client the user can call `Databases()` and get all the databases, collections, etc.

**Review by ReneWerner87 — APPROVED** [review]

**Pull request merged**